robot/impl/local_robot_test.go

```diff
@@ -4435,8 +4435,7 @@ func TestRemovingOfflineRemote(t *testing.T) {
 // prevents that behavior and removes the remote correctly.
 func TestRemovingOfflineRemotes(t *testing.T) {
```

Comment:
this test can probably be removed as the error case this tests for is no longer possible (the reconfigure and completeConfig loop interleaving), but good to make sure this doesn't come back

Reply:
I think we should leave it; it's a nice, "complicated" test involving remotes that may run through code paths we (read: me) aren't even aware of.